test/e2e/sample-e2e.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "$(dirname "$0")/../common.sh"
+source "$(dirname "$0")/setup.sh"
+
+build_sample_external_plugin
+
+export KIND_CLUSTER="kind"
```

Comment:
I do not think that will work
Instead of create a new sample-e2e.sh
You can use github actions to install the kind
Then you go into the sample dir and run go mod tidy and make, and by last, the command make test-e2e to run the tests against kind. 

is that make sense?